<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowDmm</class>
 <widget class="QMainWindow" name="MainWindowDmm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindowDmm</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>429</width>
      <height>486</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextBrowser" name="statusTextBrowser">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>400</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Measure:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="dcVoltageButton">
         <property name="text">
          <string>DC Voltage</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="dcCurrentButton">
         <property name="text">
          <string>DC Current</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="twoWireResistanceButton">
         <property name="text">
          <string>2W Resistance</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="singleReadButton">
         <property name="text">
          <string>Single Read</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="typeLabel">
         <property name="text">
          <string>Type:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="valueLabel">
         <property name="text">
          <string>Value:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>453</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
